<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>738</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QEMUTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Virtual machines</string>
    </property>
    <widget class="ConfTreeWidget" name="confTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>521</height>
      </rect>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="totalOptionsLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>731</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="optionsTabs">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>451</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="terminalPage">
     <attribute name="title">
      <string>Terminal</string>
     </attribute>
     <widget class="QPlainTextEdit" name="terminalOutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>461</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>182</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>194</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>182</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>194</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="terminalInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>481</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtTop</enum>
      </property>
      <property name="duplicatesEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Run options</string>
     </attribute>
     <widget class="QCheckBox" name="snapshotCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>-snapshot</string>
      </property>
     </widget>
     <widget class="QTableViewCheckBox" name="logOptions">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>341</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Logging options</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="logFileName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log file name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="additionalOptions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>421</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>260</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Additional run options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/settings-icon.png</normaloff>:/toolbar/icons/settings-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/Play-1-Normal-icon.png</normaloff>
     <normalon>:/toolbar/icons/Play-1-Normal-icon.png</normalon>
     <disabledoff>:/toolbar/icons/Play-1-Disabled-icon.png</disabledoff>
     <disabledon>:/toolbar/icons/Play-1-Disabled-icon.png</disabledon>
     <activeoff>:/toolbar/icons/Play-1-Hot-icon.png</activeoff>
     <activeon>:/toolbar/icons/Play-1-Hot-icon.png</activeon>
     <selectedoff>:/toolbar/icons/Play-1-Pressed-icon.png</selectedoff>
     <selectedon>:/toolbar/icons/Play-1-Pressed-icon.png</selectedon>:/toolbar/icons/Play-1-Normal-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start/play execution</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/Record-Normal-Red-icon.png</normaloff>
     <normalon>:/toolbar/icons/Record-Normal-Red-icon.png</normalon>
     <disabledoff>:/toolbar/icons/Record-Disabled-icon.png</disabledoff>
     <disabledon>:/toolbar/icons/Record-Disabled-icon.png</disabledon>
     <activeoff>:/toolbar/icons/Record-Hot-icon.png</activeoff>
     <activeon>:/toolbar/icons/Record-Hot-icon.png</activeon>
     <selectedoff>:/toolbar/icons/Record-Pressed-icon.png</selectedoff>
     <selectedon>:/toolbar/icons/Record-Pressed-icon.png</selectedon>:/toolbar/icons/Record-Normal-Red-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Record</string>
   </property>
   <property name="toolTip">
    <string>Record execution</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/add-icon.png</normaloff>:/toolbar/icons/add-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create</string>
   </property>
   <property name="toolTip">
    <string>Create new VM configuration</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/delete-icon.png</normaloff>:/toolbar/icons/delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete configuration/execution</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="qemu-gui.qrc">
     <normaloff>:/toolbar/icons/Computer-icon.png</normaloff>:/toolbar/icons/Computer-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit VM configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTableViewCheckBox</class>
   <extends>QTableView</extends>
   <header>QTableViewCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ConfTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>ConfTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qemu-gui.qrc"/>
 </resources>
 <connections/>
</ui>
